project.template.yaml

```diff
@@ -96,6 +96,1019 @@ network:
                 },
             },
             ### </Harvester compatibility special cases>
+            {
+              minmax: [5_000_000, 5_000_000],
```

Comment:
With a number this large I'd think there be a range where new types won't be updated e.g. `[5_000_000, 5_000_999]`. Not sure if thats the actual range, but can you check and bump up the max if thats the case?

It would be be nice to be able to not need to redeploy if we don't have to.

Reply:
Yes, we have such a discussion. I'll provide you the document Core team prepared to talk about this.